tests/conftest.py

```diff
@@ -70,13 +70,15 @@ def _run_semgrep_agent(
         options.append("--gitlab-json")
 
     process = subprocess.run(
-        ["python3", "-m", "semgrep_agent", *options],
+        ["semgrep-agent", *options],
```

Comment:
I think this does not work in some environments...

Reply:
can verify ^